src/core/presence.ts

```diff
@@ -123,35 +123,35 @@ export interface Presence extends EmitsDiscontinuities {
   /**
    * Method to get list of the current online users and returns the latest presence messages associated to it.
    * @param {Ably.RealtimePresenceParams} params - Parameters that control how the presence set is retrieved.
-   * @returns {Promise<PresenceMessage[]>} or upon failure, the promise will be rejected with an [[Ably.ErrorInfo]] object which explains the error.
+   * @returns {Promise<PresenceMessage[]>} or upon failure, the promise will be rejected with an {@link Ably.ErrorInfo} object which explains the error.
    */
   get(params?: Ably.RealtimePresenceParams): Promise<PresenceMember[]>;
 
   /**
    * Method to check if user with supplied clientId is online
    * @param {string} clientId - The client ID to check if it is present in the room.
-   * @returns {Promise<{boolean}>} or upon failure, the promise will be rejected with an {@link ErrorInfo} object which explains the error.
+   * @returns {Promise<{boolean}>} or upon failure, the promise will be rejected with an {@link Ably.ErrorInfo} object which explains the error.
```

Comment:
Does this break the automatic linking in typedoc? I vaguely remember having to export `ErrorInfo` in `index.ts` because otherwise it just linked to nothing

Reply:
Yeh ignore this, my IDE threw up errors and I meant to check it once the docs worked (currently broken by the rooms work), but I just checked main and it was just my IDE playing tricks.